### ISSUE TITLE
example no longer compiles on flutter 3.22.0-39.0.pre.2🐛 

### ISSUE THREAD

**Issue by TENX-S** [bdlukaa/fluent_ui#1065]
```console
../lib/src/app.dart(521,9): error GC6690633: No named parameter with the name 'checkerboardRasterCacheImages'. [D:\Developer\Projects\new\fluent_ui\example\build\windows\x64\flutter\flutter_assemble.vcxproj]
../lib/src/app.dart(551,7): error GC6690633: No named parameter with the name 'checkerboardRasterCacheImages'. [D:\Developer\Projects\new\fluent_ui\example\build\windows\x64\flutter\flutter_assemble.vcxproj]
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(254,5): error MSB8066: Custom build for 'D:\Developer\Projects\new\fluent_ui\example\build\windows\x64\CMakeFiles\3e0a05c9b9835fb56261658353644357\flutter_windows.dll.rule;D:\Developer\Projects\new\fluent_ui\example\build\windows\x64\CMakeFiles\53622a7abd104f368ed051fb40e950ca\flutter_assemble.rule;D:\Developer\Projects\new\fluent_ui\example\windows\flutter\CMakeLists.txt' exited with code 1. [D:\Developer\Projects\new\fluent_ui\example\build\windows\x64\flutter\flutter_assemble.vcxproj]
```

**Comment by bdlukaa** [comment]
We do not plan to support beta versions of Flutter. It runs on the Stable channel.

**Comment by bdlukaa** [comment]
Fixed on #1097
